src/Model/Command/UserRecommendation.php

```diff
@@ -134,6 +138,18 @@ public function setFilters(array $filters): self
         return $this;
     }
 
+    /**
+     * Specify the filter type being used.
+     */
+    public function setFilterType(string $filterType): self
+    {
+        Assertion::typeIdentifier($filterType);
```

Comment:
Also assert the allowed values:
```
Assertion::choice(
    $filterType,
    [static::FILTER_TYPE_RGX, static::FILTER_TYPE_MQL]
);
```

(like we do in `setMinimalRelevance()` etc.)